Documentation/GeneralConventions/GuidesXml.rst

```diff
@@ -0,0 +1,197 @@
+..  _guides-xml:
+
+================================
+:file:`Documentation/guides.xml`
+================================
+
+This XML file contains meta information and configuration used during rendering
+of a manual.
+
+..  hint::
+    If you are migrating from the legacy Sphinx based rendering and still have
+    a :file:`Documentation/settings.cfg` you can use an automatic migration
+    tool to :ref:`migrate the settings.cfg into a guides.xml <migrate_guides_xml>
+
+.. _settings-guides:
+
+The `<guides>` tag
+==================
+
+Is used for configuration during parsing and rendering. Most available settings
+are predefined in the rendering chain supplied by the
+:ref:`TYPO3 Rendering Container <rendering-docs>`.
+
+.. _settings-guides-project:
+
+The `<project>` tag
+===================
+
+This tag can contain the following meta information:
+
+Project title
+-------------
+
+The *project* property contains the title of the project and is displayed in the
+left sidebar (desktop) or the top (mobile) of the theme and in the title meta
+tag.
+
+For TYPO3 extensions we suggest to use the extension name in a readable
+format:
+
+*   Import / Export tool
+*   News
+*   Mask
+
+Common values are in the official TYPO3 manuals
+
+#. `<Topic> Guide`, e.g. "Frontend Localization Guide",
+   for collections of articles on a specific topic
+#. `<Topic> Reference`, e.g. "TSconfig Reference",
+   for a complete reference
+#. `<Topic> Tutorial`, e.g. "Editors Tutorial",
+   for collections of tutorials on a specific topic
+
+.. _settings-guides-version-and-release:
+
+Version and release
+-------------------
+
+The version of a manual is automatically
+
+The properties *version* and *release* both contain the version of the manual
+and mostly correspond to the version of the TYPO3 LTS or TYPO3 extension to
+which the documentation refers.
+
+The version is shown below the title in the theme's release switch and in the
+title meta tag, the release is not shown currently  - but it should be
+kept anyway to satisfy internal requirements.
+
+Normally both properties are set to the same value, either
```

Comment:
Technically, this is not correct. Sphinx distincts between those two:
https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-version